src/styles/block/nav.scss

```diff
@@ -0,0 +1,57 @@
+.nav {
+  height: 60px;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    display: flex;
+    margin-right: 20px;
+  }
+
+  &__link {
+    position: relative;
+    padding: 23px 0;
+    color: #000;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link:hover {
+    font-weight: medium;
```

Comment:
I see a similar problem in nav-bar. Whenever I move my cursor on the link - it jumps (http://joxi.ru/ZrJKWGlUbRGygm). I guess the problem is in this property, which is invalid, as dev-tool says. In case deleting it won't make any changes, I suggest you try commenting line height for this element

P.S . consider writing your scss in cascading style:
`&__link {  /*some code*/         &:hover {       /*some code*//*some code*//*some code*/   }  }`